ruby_on_rails/forms_and_authentication/project_forms.md

```diff
@@ -85,6 +111,20 @@ Now we'll start morphing our form into a full Rails form using the `#form_tag` a
 1. Test out your form.  You'll need to change your `#create` method in the controller to once again accept normal top level User attributes, so uncomment the old `User.new` line and comment out the newer one.
 1. You've just finished the first step.
 
+#### Turn Turbo back ON
+
+Above, we asked to disable Turbo for the sake of the exercise.
+
+1. Re-enable form submission with Turbo by removing the `data-turbo=false` attribute on the form tag, then also remove the hidden input with CSRF token tag and submit.
+
+   No more CSRF error!?!
+   The from is now submitted with Turbo, yet Rails still protects you by verifying a CSRF token. Where does this token comes from?
+
+1. Check your inspector and your `application.html.erb` template. See a CSRF token that s always available?
+Remove this one too from `application.html.erb`, and verify that the server hits back with a CSRF error.
```

Comment:
There is a typo here: "that s always available?".

Also, I'm not a big fan of how this looks in the markdown preview. I would either put these two lines on the same line, or add an empty line in between (line lines 118-120).

Reply:
Updated and put in a single paragraph.